README.md

```diff
@@ -189,56 +189,244 @@ Minimal required IAM permissions:
+
+1. Create an IAM policy
 
     ```bash
-    aws configure
+    IAM_POLICY_NAME=prometheus-rds-exporter
+
+    # Download policy payload
+    curl --fail --silent --write-out "Reponse code: %{response_code}\n" https://raw.githubusercontent.com/qonto/prometheus-rds-exporter/main/configs/aws/policy.json -o /tmp/prometheus-rds-exporter.policy.json
+
+    # Create IAM policy
+    aws iam create-policy --policy-name ${IAM_POLICY_NAME} --policy-document file:///tmp/prometheus-rds-exporter.policy.json
     ```
 
-2. Start application
+1. Attach IAM role to your EKS cluster
+
+    eksctl will create an IAM role and a Kubernetes service account in Kubernetes
 
     ```bash
-    prometheus-rds-exporter
+    IAM_ROLE_NAME=prometheus-rds-exporter
+    EKS_CLUSTER_NAME=default # Replace with your EKS cluster name
+    KUBERNETES_NAMESPACE=default # Replace with the namespace that you want to use
+    KUBERNETES_SERVICE_ACCOUNT_NAME=prometheus-rds-exporter
+    AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+
+    eksctl \
+    create iamserviceaccount \
+    --cluster ${EKS_CLUSTER_NAME} \
+    --namespace ${KUBERNETES_NAMESPACE} \
+    --name ${KUBERNETES_SERVICE_ACCOUNT_NAME} \
+    --role-name ${IAM_ROLE_NAME} \
+    --attach-policy-arn arn:aws:iam::${AWS_ACCOUNT_ID}:policy/${IAM_POLICY_NAME} \
+    --approve
     ```
 
-### Docker
+1. Deploy the exporter
 
-1. Connect on AWS with any method
+    ```bash
+    PROMETHEUS_RDS_EXPORTER_VERSION=0.3.0 # Replace with latest version
+    SERVICE_ACCOUNT_ANNOTATION="arn:aws:iam::${AWS_ACCOUNT_ID}:role/${IAM_ROLE_NAME}"
+
+    helm upgrade \
+    prometheus-rds-exporter \
+    oci://public.ecr.aws/qonto/prometheus-rds-exporter-chart \
+    --version ${PROMETHEUS_RDS_EXPORTER_VERSION} \
+    --install \
+    --namespace ${KUBERNETES_NAMESPACE} \
+    --set serviceAccount.annotations."eks\.amazonaws\.com\/role-arn"="${SERVICE_ACCOUNT_ANNOTATION}"
+    ```
 
-2. Start application
+1. Option. Customize prometheus exporter settings
+
+    Download Helm chart default values
 
     ```bash
-    docker run -p 9043:9043 -e AWS_PROFILE=${AWS_PROFILE} -v $HOME/.aws:/app/.aws public.ecr.aws/qonto/prometheus-rds-exporter:latest
+    helm show values oci://public.ecr.aws/qonto/prometheus-rds-exporter-chart --version ${PROMETHEUS_RDS_EXPORTER_VERSION} > values.yaml
     ```
 
-### EKS (using IRSA and Helm)
+    Customize settings
 
-1. Create an IAM policy
+    ```bash
+    vim values.yaml
+    ```
+
+    <details>
+    <summary>Example to enable debug via PROMETHEUS_RDS_EXPORTER_DEBUG environment variable</summary>
 
     ```bash
-    IAM_POLICY_NAME=prometheus-rds-exporter
-    aws iam create-policy --policy-name ${IAM_POLICY_NAME} --policy-document file://configs/aws/policy.json
+    yq --inplace '.env += {"PROMETHEUS_RDS_EXPORTER_LOG_FORMAT": "true"}' values.yaml
```

Comment:
I think it should be this:
```suggestion
    yq --inplace '.env += {"PROMETHEUS_RDS_EXPORTER_DEBUG": "true"}' values.yaml
```

Reply:
Oups, I had my doubts about what would be most relevant to the example. Debug sounds more relevant. I updated it, thanks